src/test/java/net/sf/javaanpr/recognizer/CharacterRecognizerTest.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2013 JavaANPR contributors
+ * Copyright 2006 Ondrej Martinsky
+ * Licensed under the Educational Community License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.osedu.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package net.sf.javaanpr.recognizer;
+
+import org.junit.Test;
+
+import java.util.Vector;
+
+import static org.junit.Assert.*;
+import static org.hamcrest.CoreMatchers.*;
+
+public class CharacterRecognizerTest {
+
+    private CharacterRecognizer.RecognizedChar getRecognizedCharWithThreePatterns() {
```

Comment:
Optional: This is an ok way to do create a `RecognizedChar` for testing purposed, but the way this is usually done in Junit tests is adding a setup method and then using the variable directly in the tests.

```
public class CharacterRecognizerTest {
    private RecognizedChar threePaternRecognizedChar;

    @Before
    private void setUp() {
        // ... contents of current getRecognizedCharWithThreePatterns()
    }
```